### PR TITLE
Eren helper text

### PR DESCRIPTION
**Pull request by Enmoren**

### CODE REVIEW

**Comment by coverney** [comment]
A few things:
- The "Street Address" question in the driver section has a mismatching tooltip value. The tooltip in the questions spreadsheet is "E.g. 1000 Olin Way, MA" while the one in `questions.js` is "The street address allows FMCSA to visit carriers and conduct reviews of compliance with the Federal Motor Carrier Safety Regulations and provides a crosscheck for the correct identity of the carrier. The identification number (found on the truck tractor" Can you explain what is happening here?
- Is it possible to make the dialog box for the "Manner of Crash/ Collision Impact" in the road question larger? The image is hard to parse at the current size. More generally, is it possible to resize dialog boxes based on the tooltip? Just curious

**Review by coverney — COMMENTED** [review]
A few things:
- The "Street Address" question in the driver section has a mismatching tooltip value. The tooltip in the questions spreadsheet is "E.g. 1000 Olin Way, MA" while the one in questions.js is "The street address allows FMCSA to visit carriers and conduct reviews of compliance with the Federal Motor Carrier Safety Regulations and provides a crosscheck for the correct identity of the carrier. The identification number (found on the truck tractor" Can you explain what is happening here?
- Is it possible to make the dialog box for the "Manner of Crash/ Collision Impact" in the road question larger? The image is hard to parse at the current size. More generally, is it possible to resize dialog boxes based on the tooltip? Just curious

**Comment by coverney** [comment]
@Enmoren There seems to be several tooltips in `questions.js` that aren't in the [spreadsheet](https://docs.google.com/spreadsheets/d/1OzfjUOmWOsEJ9lsy3gv8IYaJk_Xsgeb_aw3GM1flero/edit#gid=291623153). Please make sure they are consistent because then we are just hardcoding in `questions.js`, which we definitely don't want to do. Also, I would quickly proofread the tooltips. Once you did that, let me know, and I can generate a new questions.js

**Comment by Enmoren** [comment]
@coverney All the tooltips in the spreadsheet is up to dated and contains the most recent work. All the data in question.js is not important anymore. I hope this will also resolve your first and second questions.

**Comment by coverney** [comment]
> @coverney All the tooltips in the spreadsheet is up to dated and contains the most recent work. All the data in question.js is not important anymore. I hope this will also resolve your first and second questions.

Cool, I will re-generate `questions.js`, test, push, then merge

**Comment by Enmoren** [comment]
As for the img size, i don't think the modal window is resizeable based on the image size, so the trick will be how we cropped the image.

**Comment by Enmoren** [comment]
Oh I think i just pushed the lasted question.js version

**Comment by coverney** [comment]
> As for the img size, i don't think the modal window is resizeable based on the image size, so the trick will be how we cropped the image.

sounds good!

**Comment by coverney** [comment]
> Oh I think i just pushed the lasted question.js version

wait you ran `generate_questions.py`?
